frontend/src/Utils/Api/APIService.tsx

```diff
@@ -74,4 +76,40 @@ export default class APIService {
       body: JSON.stringify({ name, description, topic })
     }).catch((error) => console.error(`Error during room creation: ${error}`));
   }
+
+  static updateUser(id: number, token: string, bio: string) {
+    return fetch(`/api/update_user/${id}`, {
+      method: 'PUT',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Token ${token}`
+      },
+      body: JSON.stringify({ bio: bio })
+    }).catch((error) => console.error(`Error during updating informations: ${error}`));
+  }
+
+  static ChangeEmail(id: number, token: string, { email, email2 }: ChangeEmailConfiguration) {
```

Comment:
I suppose that email is new email, and email2 is an old email so,
`{ email, email2 }` => `{ newEmail, oldEmail }` is more readable